platform/stacks/portal.stack.yml

```diff
@@ -9,7 +9,7 @@ networks:
 services:
 
   portal:
-    image: appcelerator/portal:${TAG:-0.11.0}
+    image: appcelerator/portal:latest
```

Comment:
I don't think so, we need a way to launch the portal service for amp cluster command